### DISCUSSION TITLE
Open chat on button click

### DISCUSSION THREAD

**Discussion by Nuxor3000** [chaskiq/chaskiq#424] [Ideas]
Would it be possible to trigger an event to open the chat when a button is clicked on the website ?
Something like an entering webhook ?

**Comment by michelson** [comment]
yes, you could do 
`window.chaskiq.sendCommand('toggle');`

just assign a window.chaskiq = var to the embed script

**Reply by Nuxor3000** [reply]
I'm really sorry but I can't manage to make it work as I'm not really good with javascript.
Would you be kind to show me an example of the embed script with the "window.chaskiq = var" assigned please ?
